index.js

```diff
@@ -1,8 +1,24 @@
+var KEYBINDING_TYPES = [{
+  type: 'next',
+  func: 'moveFocusForward',
+}, {
+  type: 'prev',
+  func: 'moveFocusBack',
+}, {
+  type: 'first',
+  func: 'moveFocusToFirst',
+}, {
+  type: 'last',
+  func: 'moveFocusToLast',
+}];
+
 function FocusGroup(options) {
   options = options || {};
   this._settings = {
-    forwardArrows: options.forwardArrows || ['down'],
-    backArrows: options.backArrows || ['up'],
+    keybindings: options.keybindings || {
+      next: { keyCode: 40 },
+      prev: { keyCode: 38 },
+    },
```

Comment:
This will only provide the default options all together. What if the user wants to provide a custom `next` keybinding but accept the default `prev`? We should probably default these properties individually.